File: SCALA/Cap03/c03ex01.scala
// c03ex01.scala

object Main {
  def main(args: Array[String]): Unit = {
    var A, B, X: Int = 0

    print("Entre o 1o. valor numerico inteiro: ")
    A = scala.io.StdIn.readInt()

    print("Entre o 2o. valor numerico inteiro: ")
    B = scala.io.StdIn.readInt()

    X = A + B

    println(s"Resultado = $X")

    print("\nTecle <Enter> para encerrar... ")
    scala.io.StdIn.readLine()
  }
}

File: KOTLIN/Cap03/c03ex01.kt
// c03ex01.kt

fun main() {
    var A: Int
    var B: Int
    var X: Int

    print("Entre o 1o. valor numerico inteiro: ")
    A = readLine()?.toInt() ?: 0

    print("Entre o 2o. valor numerico inteiro: ")
    B = readLine()?.toInt() ?: 0

    X = A + B

    println("Resultado = $X")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
